tests/components/reolink/test_host.py

```diff
@@ -0,0 +1,83 @@
+"""Test the Reolink host."""
+
+from asyncio import CancelledError
+from unittest.mock import AsyncMock, MagicMock
+
+from aiohttp import ClientResponseError
+import pytest
+
+from homeassistant.components.reolink import const
+from homeassistant.components.webhook import async_handle_webhook
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.util.aiohttp import MockRequest
+
+from .conftest import TEST_UID
+
+from tests.common import MockConfigEntry
+from tests.typing import ClientSessionGenerator
+
+
+async def test_webhook_callback(
+    hass: HomeAssistant,
+    hass_client_no_auth: ClientSessionGenerator,
+    config_entry: MockConfigEntry,
+    reolink_connect: MagicMock,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test webhook callback with motion sensor."""
+    assert await hass.config_entries.async_setup(config_entry.entry_id) is True
+    await hass.async_block_till_done()
+    assert config_entry.state is ConfigEntryState.LOADED
+
+    webhook_id = f"{const.DOMAIN}_{TEST_UID.replace(':', '')}_ONVIF"
```

Comment:
Doesn't reolink use the conventional webhook_ids?

Reply:
What do you mean by "conventional webhook ids"?

The webhook id for reolink is defined here:
https://github.com/home-assistant/core/blob/66a873333326ee250084ced1e9400b639c549b8e/homeassistant/components/reolink/host.py#L558

Reply:
Usually we request a webhook id from hass which will then generate one

Reply:
Is that better and should I do that?

Reply:
I actually don't know if there is guidance or anything for this, its just something I noticed. If this were to be changed, it's definetly outside of this scope.